std/array.d

```diff
@@ -446,8 +444,9 @@ unittest
  *
  * Returns: static array of size `arr.length`
  */
-T[n] staticArray(T, size_t n)(T[n] arr)
+@nogc T[n] staticArray(T, size_t n)(T[n] arr)
 {
+    pragma(inline, true);
```

Comment:
I think it should be:
```d
pragma(inline, true)
@nogc T[n] staticArray(...
```
I haven't actually used it myself, just going from the docs.